packages/material-ui/src/Modal/Modal.test.js

```diff
@@ -285,21 +288,25 @@ describe('<Modal />', () => {
 
       expect(onEscapeKeyDownSpy).to.have.property('callCount', 1);
       expect(onCloseSpy).to.have.property('callCount', 1);
+      expect(handleKeyDown).to.have.property('callCount', 0);
     });
 
-    it('when disableEscapeKeyDown should call only onEscapeKeyDownSpy', () => {
+    it('should not call onChange when disableEscapeKeyDown', () => {
```

Comment:
```suggestion
    it('should not call onChange when `disableEscapeKeyDown=true`', () => {
```
? or something similar.